core/arch/arm/kernel/thread.c

```diff
@@ -85,8 +86,8 @@ static struct thread_core_local thread_core_local[CFG_TEE_CORE_NB_CORE];
 #ifdef ARM64
 #define STACK_CANARY_SIZE	(8 * sizeof(uint32_t))
 #endif
-#define START_CANARY_VALUE	0xdededede
-#define END_CANARY_VALUE	0xabababab
+#define START_CANARY		0xdededede
+#define END_CANARY		0xabababab
```

Comment:
Note: this change only helps lines 170 to 180 below to fit in less than 80 chars.